.gitignore

```diff
@@ -128,3 +128,6 @@ dmypy.json
 
 # Pyre type checker
 .pyre/
+
+# Test sources
+test
```

Comment:
What is this change for?

Reply:
Is not. I was testing locally on the folder and did not want to push that.